src/exceptions/erlang-otp-linking-exception.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<SPDXLicenseCollection xmlns="http://www.spdx.org/license">
+  <exception licenseId="erlang-otp-linking-exception" name="Erlang/OTP Linking Exception" listVersionAdded="3.25">
```

Comment:
The version should be 3.25.0. Since 3.24.0 we use 3 components.

Reply:
This is a new thing, so I guess the DOC was not updated. I will open a PR to update it.

Reply:
I did https://github.com/spdx/license-list-XML/pull/2528

Reply:
Sorry about that @laf0rge -- yes, still catching up on documentation updates. Thanks @xsuchy for the fix in the docs.